Simplified/Book/UI/NYPLBookDetailView.m

```diff
@@ -708,4 +721,28 @@ - (void)viewIssuesTapped {
   [self.detailViewDelegate didSelectViewIssuesForBook:self.book sender:self];
 }
 
+- (void)setUIAccessibilityFocusToView: (UIView *)view {
+  if (UIAccessibilityIsVoiceOverRunning()) {
+    [NYPLMainThreadRun asyncIfNeeded:^{
+      UIAccessibilityPostNotification(UIAccessibilityLayoutChangedNotification, view);
+    }];
+  }
+}
+
+-(void)postUIAccessibilityAnnouncementWithMessage: (NSString *)message {
+  if (UIAccessibilityIsVoiceOverRunning()) {
+    [NYPLMainThreadRun asyncIfNeeded:^{
+      UIAccessibilityPostNotification(UIAccessibilityAnnouncementNotification, message);
+    }];
+  }
+}
+
+-(void)didFinishUIAccessibilityAnnouncementNotification:(NSNotification *)notification {
+  NSString *message = notification.userInfo[UIAccessibilityAnnouncementKeyStringValue];
+  if (self.downloadStarted && [message containsString:self.normalView.messageStringForNYPLBookButtonStateSuccessful]) {
```

Comment:
also here, the `downloadStarted` check seems redundant but I may be missing something